### PR TITLE
Fix acceptance tests

### PR DESCRIPTION
**Pull request by derrxb**

### CODE REVIEW

File: spec/acceptance_tests [hunk 1]
```diff
@@ -1,5 +1,4 @@
 trap "kill 0" EXIT
 
-ruby spec/test_data.rb
```
Comment (Kyeg) [inline spec/acceptance_tests:3]: Originally, I create some new inboxes for `/inbox/12345`, `/inbox`, and `/inbox/guest-inbox` in `spec/test_data.rb`, and always load these test data before running acceptance test.

File: spec/acceptance_spec_.rb [hunk 2]
```diff
@@ -6,94 +6,96 @@
...
+    @headless = Headless.new
+    @browser = Watir::Browser.new :chrome
+
+    inbox = MindMap::Entity::Inbox.new(id: nil,
+                                       name: 'Name',
+                                       url: '12345',
+                                       description: 'Description',
+                                       suggestions: [])
+
+    MindMap::Repository::Inbox::For.klass(MindMap::Entity::Inbox).find_or_create(inbox)
```
Comment (Kyeg) [inline spec/acceptance_spec_.rb]: I think the test data here is not complete yet. 
Here is only a new inbox for `/inbox/12345`, and no new suggestions.
If we want to create test data here, maybe we should also create inboxes for `/inbox` and `/inbox/guest-inbox`.

File: spec/acceptance_tests (hunk 1 above)
Comment (derrxb) [reply spec/acceptance_tests:3]: I know, don't worry. I'll get these fix tonight. This is still a draft PR

File: spec/acceptance_spec_.rb (hunk 2 above)
Comment (derrxb) [reply spec/acceptance_spec_.rb]: Yep, I'm gonna finalize this tonight and then request a review

File: spec/acceptance_tests (hunk 1 above)
Comment (derrxb) [reply spec/acceptance_tests:3]: Ahh, I removed this because I didn't realize what it was actually doing. I move the setup into the tests themselves.

**Comment by derrxb** [comment]
Things I did in this PR:
* I removed the `test_data.rb` file since it's no longer needed.
* I ran Rubocop
* I move the test data setup into the tests that need them

In terms of merging this into master, I think I can wait until https://github.com/KED-2020/mind-map/pull/25 is merged and then I'll just fix up any merge conflicts on my end.

**Pull request closed without merging**